kv/memberlist/testlogger.go

```diff
@@ -0,0 +1,8 @@
+package memberlist
+
+type testLogger struct {
```

Comment:
Shall we move this to one of _test files?

Reply:
@pstibrany the drawback is that the definition is shared between the different test files. It's a bit weird that a shared definition is in one of them. What's the typical way of sharing code between test files? Stick a build tag in a shared .go file so it only builds for tests?

Reply:
I don't see that as a big problem. Alternatively, we can rename `testlogger.go` to `logger_test.go` so that it's clear that it is only used from tests.

Reply:
@pstibrany OK, let me see what I can do.

Reply:
Done.